### PR TITLE
suggested changes

### PR DESCRIPTION
**Pull request by jedwards4b**
These are untested as there is not yet a test procedure to use.

### CODE REVIEW

File: env_cylc.py [hunk 1]
```diff
@@ -115,33 +110,29 @@ def get_tseries_resubmit(self, ts_tper, ts_n, stop_n, stop_option):
...
         my_case = os.getcwd() + '/../'
-        
+
         case = Case(my_case, read_only=False)
-        
+
         cwd = os.getcwd()
         os.chdir(my_case)
 
-        env_case = EnvCase()
```
Comment (jedwards4b) [inline env_cylc.py]: All of these get_value arguments are available from case, no need to create objects for each file interface.

File: env_cylc.py [hunk 2]
```diff
@@ -115,33 +110,29 @@ def get_tseries_resubmit(self, ts_tper, ts_n, stop_n, stop_option):
...
         print 'Running on ',machine_name
 
-        env_mach = Machines(machine=machine_name)
-        batch_system = env_mach.get_value("BATCH_SYSTEM")
+        batch_system = case.get_value("BATCH_SYSTEM")
         batch = Batch(batch_system=batch_system, machine=machine_name)
-        env_run = EnvRun()
-        env_build = EnvBuild() 
         env_batch = case.get_env("batch")
         os.system('./xmlchange RESUBMIT=0')
```
Comment (jedwards4b) [inline env_cylc.py]: You have a case object, instead of this os.system call you can do 
case.set_value(RESUBMIT, 0)

File: env_cylc.py [hunk 3]
```diff
@@ -192,25 +183,25 @@ def get_env(self):
...
-        self.env['USER'] = env_case.get_value('USER')
+        self.env['STOP_N'] = case.get_value("STOP_N")
+        self.env['RESUBMIT'] = case.get_value("RESUBMIT")
+        self.env['STOP_OPTION'] = case.get_value('STOP_OPTION')
+        self.env['DOUT_S'] = case.get_value('DOUT_S')
+        self.env['DOUT_L_MS'] = case.get_value('DOUT_L_MS')
+        self.env['CASEROOT'] = case.get_value('CASEROOT')
+        self.env['CASE'] = case.get_value('CASE')
+        self.env['RUNDIR'] = case.get_value('RUNDIR')
+        self.env['CIME_OUTPUT_ROOT'] = case.get_value('CIME_OUTPUT_ROOT')
```
Comment (jedwards4b) [inline env_cylc.py]: CIME_OUTPUT_ROOT replaces CESMSCRATCHROOT

**Comment by sherimickelson** [comment]
Closing this PR as it was added in through another PR.
Some of this required hand tweeking to merge and required other updates.

**Pull request closed without merging**